File: BAEKJOON/cpp/3449_해밍_거리.cpp
#include <iostream>
#include <string>
using namespace std;

string a, b;

void input() {
	cin >> a >> b;
}

void solution() {
	int answer = 0;

	for (int i = 0; i < a.length(); i++) {
		if (a[i] != b[i]) {
			answer++;
		}
	}

	cout << "Hamming distance is " << answer << ".\n";
}

int main() {
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);
	
	int tc;
	cin >> tc;

	for (int i = 0; i < tc; i++) {
		input();
		solution();
	}

	return 0;
}

File: BAEKJOON/java/3449_해밍_거리.java
import java.io.*;

public class Main {
    public static String a, b;

    public static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

    public static void main(String[] args) throws IOException {
        int tc = Integer.parseInt(br.readLine());

        for (int i = 0; i < tc; i++) {
            input();
            solution();
        }

        bw.flush();
        bw.close();
    }

    private static void input() throws IOException {
        a = br.readLine();
        b = br.readLine();
    }

    private static void solution() throws IOException {
        int answer = 0;

        for (int i = 0; i < a.length(); i++) {
            if (a.charAt(i) != b.charAt(i)) {
                answer++;
            }
        }

        bw.write("Hamming distance is " + answer + ".\n");
    }
}